pandapipes/converter/stanet/stanet2pandapipes.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) 2020-2022 by Fraunhofer Institute for Energy Economics
+# and Energy System Technology (IEE), Kassel, and University of Kassel. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+import numpy as np
+import pandas as pd
+
+from pandapipes.converter.stanet.preparing_steps import get_net_params, get_pipe_geo, \
+    connection_pipe_section_table, get_stanet_raw_data, create_meter_table, create_house_table
+from pandapipes.converter.stanet.table_creation import create_junctions_from_nodes, \
+    create_valve_and_pipe, create_pumps, create_junctions_from_connections, \
+    create_pipes_from_connections, create_heat_exchanger_sta, create_slider_valves, \
+    create_pipes_from_remaining_pipe_table, create_nodes_house_connections, \
+    create_sinks_meters, create_sinks_from_nodes, create_control_components, \
+    create_sinks_from_customers, create_pipes_house_connections
+from pandapipes.create import create_empty_network
+
+try:
+    import pandaplan.core.pplog as logging
+except ImportError:
+    import logging
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: Some things are left to do for this converter:
+#         - in case of house connection calculation in STANET, the connections have results VMA and
+#           VMB. These results can be used for comparison, but also for sorting of connections on a
+#           main pipe --> especially if connection nodes on the main pipe have the same coordinates.
+#         - some TODOs in table_creation
+#         - maybe it will be necessary to remove deleted data from the STANET tables, otherwise they
+#           might be inserted into the pandapipes net erroneously
```

Comment:
despite the opne todos I think we should merge the PR. Maybe add some infos. What do yo think, @dlohmeier

Reply:
For the first version, this should be fine.